### DISCUSSION TITLE
How do you access `(include)` widgets and windows?

### DISCUSSION THREAD

**Discussion by max-ishere** [elkowar/eww#981] [Q&A] [answered]
I have created this:

```txt
.config/eww/
  eww.yuck
  modules/
    test.yuck
```

But I cant open any windows defined in `test.yuck`. The actual file structure is a bit more complex, so I'm interested in how the include statement names the windows and widgets.

--------

My idea is to make a package manager that allows you to separate the widgets into different files and import the ones you like and basically kitbash your own bar for example out of different modules. But it could also be a dashboard too. Anything goes.

**Comment by w-lfchen** [comment, marked as answer]
I feel like there is something wrong with your `include` declaration.
If I have the following structure:
```
.
|- eww.yuck
|- menu/
 |- menu.yuck
```
and then use `(include "./menu/menu.yuck")` in the `eww.yuck` file, I am able to open the window `menu`, declared in `menu.yuck`, simply using `eww open menu`.

**Reply by max-ishere** [reply]
~~Can I nest includes inside included files?~~

Ok, I've looked into it and I forgot to include the one file that has the windows. I just had the widget file included.

However it seems that all includes are absolute which is a bummer because I really wish they were relative instead.
